webapp/src/components/backstage/playbook_runs/playbook_run_backstage/overview/description.tsx

```diff
@@ -26,12 +29,15 @@ const StyledContent = styled(Content)`
     padding: 20px 24px 14px 24px;
 `;
 
+interface MatchParams {
+    playbookRunId: string
+}
+
 const Description = (props: { playbookRun: PlaybookRun }) => {
     const {formatMessage} = useIntl();
     const team = useSelector<GlobalState, Team>((state) => getTeam(state, props.playbookRun.team_id));
-
     let summary: JSX.Element = <EmptyBody>{formatMessage({defaultMessage: 'There is no run summary available.'})}</EmptyBody>;
-    if (props.playbookRun.status_posts.length > 0 && props.playbookRun.summary) {
+    if (props.playbookRun.status_posts.length >= 0 && props.playbookRun.summary) {
```

Comment:
Could you elaborate on this change?

Reply:
Hi @iomodo. I saw that Summary(description) was not being displayed when there we no status posts. So, i did this change.

Reply:
Then probably it should be:

```suggestion
    if (props.playbookRun.summary) {
```

Reply:
Absolutely Right! A Face Palm moment for me :man_facepalming: